bot/exts/evergreen/githubinfo.py

```diff
@@ -23,27 +25,28 @@ async def fetch_data(self, url: str) -> dict:
         async with self.bot.http_session.get(url) as r:
             return await r.json()
 
-    @commands.command(name='github', aliases=['gh'])
-    @commands.cooldown(1, 60, BucketType.user)
-    async def get_github_info(self, ctx: commands.Context, username: Optional[str]) -> None:
-        """
-        Fetches a user's GitHub information.
-
-        Username is optional and sends the help command if not specified.
-        """
-        if username is None:
-            await ctx.invoke(self.bot.get_command('help'), 'github')
-            ctx.command.reset_cooldown(ctx)
-            return
+    @commands.group(name='github', aliases=('gh', 'git'))
+    @commands.cooldown(1, 10, BucketType.user)
+    async def github_group(self, ctx: commands.Context) -> None:
+        """Commands for finding information related to GitHub."""
+        if ctx.invoked_subcommand is None:
+            await invoke_help_command(ctx)
 
+    @github_group.command(name='user', aliases=('userinfo',))
+    async def github_user_info(self, ctx: commands.Context, username: str) -> None:
+        """Fetches a user's GitHub information."""
         async with ctx.typing():
-            user_data = await self.fetch_data(f"https://api.github.com/users/{username}")
+            user_data = await self.fetch_data(f"{GITHUB_API_URL}/users/{username}")
```

Comment:
Should we url quote the `username` here before appending it to the url and fetching?